probe_src/python/probe_py/manual/remote_access.py

```diff
@@ -65,8 +65,8 @@ def copy_provenance(source: HostPath, destination: HostPath) -> None:
 ProvenanceInfo: typing.TypeAlias = tuple[
     list[InodeVersion],
     list[InodeMetadata],
-    typing.Mapping[int, Process],
-    typing.Mapping[InodeVersion, int | None],
+    typing.Dict[int, Process],
+    typing.Dict[InodeVersion, int | None],
```

Comment:
```suggestion
    dict[int, Process],
    dict[InodeVersion, int | None],
```

`typing.Mapping[K, V]` is immutable; `dict[K, V]` is mutable. `typing.Dict[K, V]` is a longer way of saying `dict`.